src/pyedb/dotnet/edb_core/components.py

```diff
@@ -2426,6 +2426,11 @@ def get_pins_name_from_net(self, pin_list, net_name):
 
         """
         pinlist = []
```

Comment:
It is a bit confusing to see both `pinlist` and `pin_list` in the same method.
Would it make sense to use `pins_name` or `res` instead of `pinlist` ?